instance/models/instance.py

```diff
@@ -547,6 +611,48 @@ def studio_url(self):
         """
         return u'{0.protocol}://{0.studio_domain}/'.format(self)
 
+    @property
+    def database_name(self):
+        """
+        The database name used for external databases
+        """
+        return self.domain.replace('.', '_')
```

Comment:
@omarkhan Wouldn't that require more sanitizing? For example, to take into account the maximum size of the database name (64 for MySQL for example)? And are there other characters than '.' that might be allowed in domain names, but not in database names?